lib/rubocop/cop/mixin/def_node.rb

```diff
@@ -21,6 +21,10 @@ def preceding_non_public_modifier?(node)
         end
       end
 
+      def stripped_source_upto(index)
+        processed_source[0..index].map(&:strip)
+      end
+
```

Comment:
This was originally in `util.rb`. But this mixin is the only place the method is used. So here it is.